src/plugin.py

```diff
@@ -30,6 +30,28 @@
 
 
 class BNetPlugin(Plugin):
+    async def get_game_time(self, game_id, context):
+        gameTimeMinutes = 0
+        log.info('game_id: ' + game_id)
+        if game_id == "5272175": # Overwatch
+            log.debug("Fetching playtime for Overwatch...")
+            player_data = await self.backend_client.get_ow_player_data()
+            if player_data == None:
```

Comment:
recommended None comparison 
```suggestion
            if player_data is None:
```